src/duckling/main.reducer.ts

```diff
@@ -4,6 +4,7 @@ import {selectionReducer, copyPasteReducer} from './selection';
 import {Action} from './state';
 import {optionsReducer} from './state/options.service';
 import {layerReducer} from './entitysystem/services/entity-layer.service';
+import {collisionTypesReducer} from './game/collision/collision-types.service';
```

Comment:
THIS IS BAD. We currently don't have a way of adding in reducers in game specific logic without breaking our golden rule.